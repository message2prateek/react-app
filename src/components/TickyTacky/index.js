import * as React from 'react'

import Square from '../square'

export default function TickyTacky () {
  return (
    <>
      <h1>Ticky Tacky</h1>
      <Square>x</Square>
    </>
  )
}
