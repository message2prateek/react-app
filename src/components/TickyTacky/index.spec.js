import * as React from 'react'

import TickyTacky from './'
import { shallow } from 'enzyme'

describe('TickyTacky', () => {
  it('matches the snapshot', () => {
    expect(toJson(shallow(<TickyTacky />))).toMatchSnapshot()
  })
})
