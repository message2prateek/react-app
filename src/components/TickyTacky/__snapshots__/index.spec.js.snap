// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TickyTacky matches the snapshot 1`] = `
<h1>
  Ticky Tacky
</h1>
`;
